### PR TITLE
soc: smartbond: Global Foundries support

### PR DESCRIPTION
**Pull request by ioannis-karachalios**
Add the min. SoC changes to support the GF silicon variant.

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -117,7 +117,7 @@ manifest:
         - hal
     - name: hal_renesas
       path: modules/hal/renesas
-      revision: f2d791d28cd8fdbc5861652b863822632c91f690
+      revision: pull/1/head
```
Comment (andrzej-kaczmarek) [inline west.yml:120]: should be `pull/3/head`

Comment (PROFESSOR-J-K) [reply west.yml]: Done.  May I ask how this line should be interpreted?

Comment (andrzej-kaczmarek) [reply west.yml]: this is github ref that refers to head commit of PR and I believe you need PR#3 from hal_renesas for this PR

File: soc/arm/renesas_smartbond/da1469x/soc.c [hunk 2]
```diff
@@ -25,10 +31,41 @@ static int renesas_da14699_init(void)
...
+		issue a fake event. This is typically observed upon power sources 
+		switching, that is DCDC <--> LDOs <--> Retention LDOs
+     */
+	CRG_TOP->BOD_CTRL_REG &= ~(CRG_TOP_BOD_CTRL_REG_BOD_V14_EN_Msk  |
+	                           CRG_TOP_BOD_CTRL_REG_BOD_V18F_EN_Msk |
+							   CRG_TOP_BOD_CTRL_REG_BOD_VDD_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V18P_EN_Msk |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V18_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_V30_EN_Msk  |
+							   CRG_TOP_BOD_CTRL_REG_BOD_VBAT_EN_Msk);
```
Comment (andrzej-kaczmarek) [inline soc/arm/renesas_smartbond/da1469x/soc.c:53]: indentation in added lines is broken

**Review by andrzej-kaczmarek — CHANGES_REQUESTED** [review]
please fix commit message, it's hard to read and the prefix should include smartbond

File: soc/arm/renesas_smartbond/da1469x/soc.c (hunk 2 above)
Comment (ioannis-karachalios) [reply soc/arm/renesas_smartbond/da1469x/soc.c:53]: To be honest, I do not know how to fix this and why indentation has been broken.

Comment (ioannis-karachalios) [reply soc/arm/renesas_smartbond/da1469x/soc.c]: I think it should be OK now.

**Review by andrzej-kaczmarek — DISMISSED** [review]

**Review by andrzej-kaczmarek — DISMISSED** [review]

**Review by ydamigos — APPROVED** [review]

**Comment by andrzej-kaczmarek** [comment]
@ioannis-karachalios please rebase

File: west.yml
Comment (andrzej-kaczmarek) [inline west.yml:1]: seems like original commit was squashed with some commit that is not related to this change

**Review by andrzej-kaczmarek — CHANGES_REQUESTED** [review]

File: west.yml
Comment (ioannis-karachalios) [reply west.yml:1]: I do not know how this happened. To be honest I do not know how this could be fixed.

Comment (ioannis-karachalios) [reply west.yml:1]: Fixed

**Review by andrzej-kaczmarek — DISMISSED** [review]

**Review by andrzej-kaczmarek — DISMISSED** [review]

**Review by carlescufi — DISMISSED** [review]

**Review by andrzej-kaczmarek — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]
